### ISSUE TITLE
Add logic to the security wrappers activation script removing old wrapper paths if they exist

### ISSUE THREAD

**Issue by ixmatus** [NixOS/nixpkgs#24164]
## Issue description

This is specifically in reference to #23641 which addresses a regression I introduced in another PR adding setcap wrapper support with additional logic for cleaning up the legacy wrapper paths; at the time I hadn't thoroughly thought through nor tested the migration scenarios until a user experienced the problem themselves.

We went ahead and merged #23641 to unblock 17.03 and @abbradar had a good suggestion for discriminating on the active current system when checking for the legacy wrapper paths to be removed and removing them.

This ticket tracks the work we need to do _after **17.03** is released_ to cleanup the legacy wrapper paths safely.

**Comment by mmahut** [comment]
This has been merged in.
